Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM alpine
```

Comment:
Can we use gcr.io/distroless/static:latest?  alpine is not an image that k8s encourages

Reply:
sure, update https://github.com/kubernetes-csi/csi-driver-host-path/blob/master/Dockerfile too?

Reply:
Yes we should look into how to get hostpath off of alpine too

Reply:
Done

Reply:
The problem is that CSI drivers may need additional tools like mkfs and fsck. I doubt that those are available in the distroless image, but I'd be happy to be proven wrong.

Reply:
It's hard to predict exactly all the tools/packages flex drivers will need.

We could use a k8s debian image instead which will have some of the basic filesystem tools.  Or we may have to document that you may need to change the base image based off of your driver's requirements.

Reply:
How about using flexadapter main docker image to distribute flexadapter binary only? Different flex storage implementations may require their own packages, e.g. nfs-utils, etc. 

Our [simple-nfs dockerfile](https://github.com/kubernetes-csi/csi-driver-flex/blob/master/examples/simplenfs-flexdriver/dockerfile/Dockerfile) can be implemented like this:

```
FROM flexadapter:latest as adapter

FROM centos:7.4.1708

# Copy flexadapter from flexadapter image
COPY --from=adapter flexadapter /flexadapter
# Copy nfs from driver directory
COPY nfs /drivers/nfs

RUN yum -y install nfs-utils && yum -y install epel-release && yum -y install jq && yum clean all 

ENTRYPOINT ["/flexadapter", "--driverpath=/drivers/nfs"]
```

Reply:
sgtm, we can update the documentation/examples to reflect this